.eslintrc.js

```diff
@@ -38,76 +43,79 @@ module.exports = {
     {
       files: ['*.ts'],
       extends: ['@metamask/eslint-config-typescript'],
+      parserOptions: {
+        project: ['./tsconfig.packages.json'],
```

Comment:
These docs seem applicable: https://typescript-eslint.io/linting/typed-linting/monorepos

This seems correct except that they also advise setting `tsconfigRootDir`

```suggestion
        tsconfigRootDir: __dirname,
        project: ['./tsconfig.packages.json'],
```

I'm not entirely sure what the consequences are of omitting it (it seems to work for me either way).